src/config/ident.rs

```diff
@@ -9,6 +10,15 @@ pub enum Ident {
     Pattern(Regex),
 }
 
+impl fmt::Display for Ident {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match *self {
+            Ident::Name(ref name) => write!(f, "{}", name),
+            Ident::Pattern(_) => write!(f, "Regex"),
```

Comment:
Should we maybe store the regex pattern string here so that it can be printed? Seems useful

Reply:
I can add a todo if you want? For now, it's not used so...

Reply:
Sure